int16 left_rpm
int16 right_rpm
uint32 left_ticks
uint32 right_ticks
